readme/dev/accessibility.md

```diff
@@ -0,0 +1,91 @@
+    - See also [NVDA browse and focus modes](https://download.nvaccess.org/documentation/userGuide.html#BrowseMode), [Orca: Filling out forms (discusses browse and focus modes)](https://help.gnome.org/users/orca/stable/howto_forms.html.en).
+
+### Mobile: Setting up a screen reader
+
+Like desktop, most physical Android and iOS devices come with screen readers. However, setting up a screen reader on an Android emulator or iOS simulator can be a bit more difficult:
+- **Physical Android device**:
+	- Android devices usually come with [TalkBack](https://support.google.com/accessibility/android/answer/6007100?hl%3Den#).
+	- See `developer.android.com`'s [guide for testing with TalkBack](https://developer.android.com/guide/topics/ui/accessibility/testing#talkback).
+- **Physical iOS device**:
+	- iOS comes with [VoiceOver](https://support.apple.com/guide/iphone/turn-on-and-practice-voiceover-iph3e2e415f/ios).
+- **iOS simulator**:
+	1. Start the simulator.
+	2. Enable [VoiceOver for MacOS](https://support.apple.com/guide/voiceover/get-started-vo4be8816d70/mac).
+	3. Focus the simulator's window.
+	4. Press <kbd>control</kbd>-<kbd>option</kbd>-<kbd>rightArrow</kbd> repeatedly to move VoiceOver focus into the simulator.
+		- **Note**: Sometimes, VoiceOver reads "Simulator not responding" before allowing VoiceOver focus to be moved into the simulator.
+- **Android emulator**:
+	- On Android, it may be necessary to install TalkBack from the Google Play Store (in an emulator with the Play Store installed) or [build a TalkBack APK from source](https://github.com/google/talkback).
+	- After installation, TalkBack can be enabled as on a physical Android device, from the settings app.
+- **Web**:
+	- Enable the operating system's screen reader (see "Desktop: Setting up a screen reader").
+
+On a physical mobile device, VoiceOver and TalkBack share certain gestures:
+- **To focus the next item**: Swipe from left to right.
+- **To focus the previous item**: Swipe from right to left.
+- **To jump focus to a particular part of the screen**: Tap once on the part of the screen to focus.
+- **To click on the focused item**: Double-tap anywhere on the screen.
+
+## Adding automated tests to prevent regressions
+
+The desktop and mobile apps use different frameworks for checking for accessibility-related regressions: The desktop app uses [Playwright](https://playwright.dev/) and the mobile app uses [react-native-testing-library](https://callstack.github.io/react-native-testing-library/).
+
+### Desktop: Automated accessibility testing
+
+Existing Playwright automated tests can be found in `packages/app-desktop/integration-tests`. These tests can be run with `yarn test-ui` from `packages/app-desktop`. It's possible to run a subset of all tests using `yarn test-ui -g "test name search pattern"`, where `-g` stands for "grep".
```

Comment:
As I understand Playwrite is more generally for our UI tests. In that case is there a document in our own documentation we can link to?

Reply:
I've moved some of this information to `app-desktop/integration-tests/README.md`. In the future, it may make sense to move this README to the `readme/dev` folder.